### ISSUE TITLE
The status bar components doesn't keep the same position after reboot

### ISSUE THREAD

**Issue by qeude** [gao-sun/eul#40]
If you put change the position of the status bar components then restart your Mac, then the status bar components goes at the extreme left of the status bar.
Which involve hiding it with app as [Dozer.](https://github.com/Mortennn/Dozer)

**Comment by gao-sun** [comment]
thanks for reporting, will take a look before next release

**Comment by gao-sun** [comment]
cannot reproduce in Big Sur, would you mind to provide your device and OS info?

**Comment by qeude** [comment]
I experienced this bug using eul on macOS Catalina v10.15.7 on an iMac 27" 2019.
I'm running eul 1.3.1, and experienced this today.
As I said, I'm using [Dozer.](https://github.com/Mortennn/Dozer) which may be the problem.

**Comment by gao-sun** [comment]
Yeah I tried to hide eul using Dozer and logout/login but cannot reproduce - will try on Catalina if possible

**Comment by YeungKC** [comment]
When I update the widget position it also refreshes the position.

**Comment by jbwzrd213** [comment]
Not sure if this is the same issue, but the position always resets every time I run the app (to the farthest left).

OS: 10.15.7 Catalina
Mid-2014 15" MacBook Pro
2.5 GHz Quad-Core Intel Core i7

**Comment by gao-sun** [comment]
sounds strange. did not have the same experience when I was debugging in Catalina. will try to reproduce these days

**Comment by gao-sun** [comment]
guys sorry i totally misunderstood the point of this issue, so i was on the wrong track. :-)

please try v1.4.8, it should be good in Catalina now. I don't have Catalina MacBook handy so please let me know if it works. unfortunately it seems Big Sur has [a bug](https://github.com/feedback-assistant/reports/issues/151) on this so if you are on it you need to wait the fix from Apple

**Comment by MeisterLampeMK** [comment]
same issue under Version 1.5 in Catalina

**Comment by MeisterLampeMK** [comment]
@gao-sun when is the issue fixed?
